packages/google-cloud-run/src/index.js

```diff
@@ -4,10 +4,13 @@
 
 'use strict';
 
+const path = require('path');
 // MAINTENANCE NOTE: All code in this file needs to be compatible with all Node.js versions >= 6.0.0.
 
 try {
   const nodeJsVersion = process.version;
+  // eslint-disable-next-line no-console
+  console.log('@instana/google-cloud-run module version:', require(path.join(__dirname, '..', 'package.json')).version);
```

Comment:
_nitpick:_ this should be above line 11, I think. The `const nodeJsVersion = process.version;` and the `if (typeof nodeJsVersion !== 'string') {` belong together and now this sits right in the middle between them.

Reply:
Resolved the comment